internal/internal_test.go

```diff
@@ -171,52 +171,6 @@ func TestRandomSleep(t *testing.T) {
 	assert.True(t, elapsed < time.Millisecond*150)
 }
 
-func TestDuration(t *testing.T) {
```

Comment:
do these need to be moved over to the config.Duration tests or are they duplicated? I can't remember.

Reply:
Oh I was assuming it's there but it isn't will fix.